Console/Microsoft.DataTransfer.ConsoleHost/Configuration/CommandLineConfiguration.cs

```diff
@@ -59,9 +59,21 @@ public static CommandLineConfiguration Parse(string[] arguments)
             if (sourceConfiguration.TryGetValue(String.Empty, out sourceName))
                 sourceConfiguration.Remove(String.Empty);
 
+            if (sourceConfiguration.TryGetValue("ConnectionString", out sourceName) && sourceName.EndsWith("/"))
```

Comment:
This file is only to parse shared configuration. ConnectionString is specific to source/target, therefore should be handled in the appropriate source/target project, not here.

Reply:
okay, I've moved it from this file to the appropriate source/sink adapter code.